include/item_menu_icons.h

```diff
@@ -19,7 +19,13 @@ void sub_80D4FC8(u8 arg0);
 void sub_80D4FEC(u8 arg0);
 u8 CreateBerryTagSprite(u8 id, s16 x, s16 y);
 void FreeBerryTagSpritePalette(void);
-u8 sub_80D511C(u8 berryId, u8 x, u8 y, bool8 startAffine);
+u8 LoadSpinningBerryPicGfx(u8 berryId, u8 x, u8 y, bool8 startAffine);
 u8 CreateBerryFlavorCircleSprite(s16 x);
 
+#define BAG_GFX 100
```

Comment:
These 5 defines are graphics tags, so they should all have a `TAG_` prefix in their names.  (i.e. `TAG_ROTATING_BALL`, TAG_BERRY_PIC`, etc.)